oasislmf/model_preparation/gul_inputs.py

```diff
@@ -186,6 +188,7 @@ def get_gul_input_items(
         # keys dataframes on loc. number/loc. ID; filter out any rows with
         # zeros for TIVs for all coverage types, and replace any nulls in the
         # cond.num. and TIV columns with zeros
+        exposure_df[SOURCE_IDX['loc']] = exposure_df.index.values
```

Comment:
There's no need to specifically assign the index values (a Numpy array) - the index will do, e.g.

    exposure_df[SOURCE_IDX['loc']] = exposure_df.index